.github/workflows/backend-test.yml

```diff
@@ -0,0 +1,95 @@
+name: Backend Test
+
+on:
+  push:
+  workflow_dispatch:
+
+jobs:
+  misc:
+    runs-on: ubuntu-latest
+    env:
+      CMAKE_DUMMY_WEBUI: true
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+      - uses: actions/setup-python@v2
+      - run: python -m pip install -r rst/requirements.txt
+
+      - name: Install tarantool
+        run: |
+          curl -L https://tarantool.io/installer.sh | sudo VER=1.10 bash
+
+      - name: Install ldoc
+        run: >
+          tarantoolctl rocks install --server=http://rocks.moonscript.org
+          https://raw.githubusercontent.com/tarantool/ldoc/tarantool/ldoc-scm-2.rockspec
+
+      - name: Build cartridge
+        env:
+          CMAKE_LDOC_FIND_REQUIRED: 'YES'
+          CMAKE_SPHINX_FIND_REQUIRED: 'YES'
+        run: |
+          tarantoolctl rocks make
+
+      - name: Lint
+        run: |
+          tarantoolctl rocks install luacheck
+          .rocks/bin/luacheck .
+
+      - name: Check schema
+        run: |
+          npm install graphql-cli@3.0.14
+          ./fetch-schema.sh
```

Comment:
I would rather refactor steps to:

1. Install all dependencies
2. lint code
3. tarantoolctl rocks make
3. ./fetch-schema.sh

Reply:
Do you want to swap linting with rocks make? What would it change?

Reply:
It will give you feedback faster, when it is linting error

Reply:
and linting is quick